File: treessence/src/main/java/fr/bipi/tressence/formatter/SimpleFormatter.java
package fr.bipi.tressence.formatter;

public class SimpleFormatter implements Formatter {

    public static final SimpleFormatter INSTANCE = new SimpleFormatter();

    private SimpleFormatter() {
    }

    @Override
    public String format(int priority, String tag, String message) {
        return tag + " : " + message;
    }
}

File: treessence/src/main/java/fr/bipi/treessence/common/formatter/SimpleFormatter.kt
package fr.bipi.treessence.common.formatter

class SimpleFormatter private constructor() : Formatter {
    override fun format(priority: Int, tag: String?, message: String): String {
        return "${tag ?: ""} : ${message}\n"
    }

    companion object {
        val INSTANCE = SimpleFormatter()
    }
}
